pennylane/capture/primitives.py

```diff
@@ -318,3 +343,32 @@ def _(*args, has_eigvals=False, **_):
         return abstract_type(abstract_eval, n_wires=n_wires, has_eigvals=has_eigvals)
 
     return primitive
+
+
+@lru_cache
+def create_grad_primitive():
```

Comment:
Make `get` instead of `create`? Since most of the time we're just retrieving one that's already been created.

Reply:
Ah, yeah good point!
- I was following the naming of the other functions in this file.
- I suppose within PL itself we only use this function once, and then it is creating the primitive...
That being said, I don't have a strong preference. Should I change?